ERCS/erc-7653.md

```diff
@@ -0,0 +1,96 @@
+---
+eip: 7653
+title: Named NFT
+description: An extension to ERC-721 based on ERC-7632.
+author: Zainan Victor Zhou (@xinbenlv)
+discussions-to: https://ethereum-magicians.org/t/erc-7653-named-nfts-erc-721-extension/18550
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-03-13
+requires: 165, 721, 7632
+---
+
+## Abstract
+
+An extension to `ERC-721` based on `ERC-7632` to enable named `ERC-721`
```

Comment:
Your abstract is certainly terse, but it doesn't contain enough information about what the proposal actually does. What is a named NFT? How do the names work?